src/components/v1/V1Dashboard.tsx

```diff
@@ -96,13 +92,9 @@ export default function V1Dashboard() {
   const uri = useUriOfProject(projectId)
   const { data: metadata } = useProjectMetadata(uri)
 
-  useEffect(() => {
-    if (metadata?.name) {
-      document.title = `${metadata.name} | Juicebox`
-    } else {
-      document.title = DEFAULT_SITE_TITLE
-    }
-  }, [metadata])
+  usePageTitle({
```

Comment:
nit:
I wonder if we should roll an auxilary hook like:
```ts
useJuiceboxPageTitle = (title?: string) => usePageTitle({title: title? `${title} | Juicebox` : undefined})
```
sorta psuedocode - but you get the idea?

Reply:
Yeah good suggestion for sure, might leave for now though. Can def revisit.